contracts/src/v0.8/automation/dev/MercuryRegistry.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity 0.8.6;
 import {ConfirmedOwner} from "../../shared/access/ConfirmedOwner.sol";
 import {AutomationCompatibleInterface} from "../interfaces/AutomationCompatibleInterface.sol";
 import {StreamsLookupCompatibleInterface} from "../interfaces/StreamsLookupCompatibleInterface.sol";
-import {ChainSpecificUtil} from "../../ChainSpecificUtil.sol";
+import {ChainSpecificUtil} from "../../ChainSpecificUtil_v0_8_6.sol";
```

Comment:
Is this safe, we have to notify the automation team about this? They need to review this PR as well.

Reply:
yes good point. let me ping their channel